smack-integration-test/src/main/java/org/jivesoftware/smackx/muc/MultiUserChatIntegrationTest.java

```diff
@@ -161,4 +695,29 @@ public void roomDestroyed(MultiUserChat alternateMUC, String reason) {
         assertEquals(0, muc.getOccupantsCount());
         assertNull(muc.getNickname());
     }
+
+    /**
+     * Gets a random room name
+     *
+     * @param prefix A prefix to add to the room name for descriptive purposes
+     */
+    private EntityBareJid getRandomRoom(String prefix) throws XmppStringprepException {
+        final String roomNameLocal = prefix + StringUtils.insecureRandomString(6);
```

Comment:
I know that this was not strictly introduced by you, but while at it: Objects created during a sinttest run should carry the `testRunId`, which is a field of `AbstractSmackIntTest`. Hence this should probably be

```
final String roomNameLocal = prefix + "-" + testRunId + "-" StringUtils.insecureRandomString(6);
```

Note that I added the dash after prefix, which means call sites of this function must remove it. For example
`getRandomRoom("smack-inttest-join-leave-")`
becomes
`getRandomRoom("smack-inttest-join-leave")`